docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Comments for YAML files start with #, actual comments cause error when executing.